cygnus-common/pom.xml

```diff
@@ -16,6 +16,12 @@
     </properties>
 
     <dependencies>
```

Comment:
Weird indent in the added XML fragments...

Reply:
fixed